### DISCUSSION TITLE
Extracting Metadata like as Regex and Entity Recognition of IPED Base

### DISCUSSION THREAD

**Discussion by mcquadenc** [sepinf-inc/IPED#2140] [General]
Hello,

I need to extract all regex and recognized entity-related information applied on a solr/iped basis, not specifically for a file, but in general, even if it has the related files.

Someone already extract this data by command line, like python or java?

Is there some sample to this context. 

I tested Web API and Some tests using python jnius for another case, but i cannot found a sample for this case. }

Ps.: I would like to help to maintain and improve this API Rest feature.

**Comment by lfcnassif** [comment]
Hi @mcquadenc,

Did you see this? https://github.com/sepinf-inc/IPED/discussions/875

You should just call `item.getExtraAttribute("Regex:CPF")` instead of `item.getName()`

> Ps.: I would like to help to maintain and improve this API Rest feature.

That would be very welcome because the original contributor of the Rest API no longer works in our agency neither with forensics and I considered a few times to remove that web api because of lack of human resources to maintain or simply test it between releases.

**Reply by lfcnassif** [reply]
Just updated the script on #875 to work with 4.x versions.

**Reply by lfcnassif** [reply]
See also https://github.com/sepinf-inc/IPED/discussions/701#discussioncomment-1190875

**Reply by mcquadenc** [reply]
I am sorry for late. I had been traveling. 
Thank you! I noticed the difference between versions. After that, I could fix it, but Linux already has a problem with binary dependence.  

I will try using Windows, It will be easier to manage it. 
After that, I return feedback here.
